ios/RNZoomUs.m

```diff
@@ -129,9 +125,14 @@ - (dispatch_queue_t)methodQueue
       ms.delegate = self;
 
       MobileRTCMeetingJoinParam * joinParam = [[MobileRTCMeetingJoinParam alloc]init];
-      joinParam.userName = displayName;
-      joinParam.meetingNumber = meetingNo;
-//       joinParam.password = pwd;
+      joinParam.userName = data[@"userName"];
+      joinParam.meetingNumber = data[@"meetingNumber"];
+      joinParam.password =  data[@"password"];
+      joinParam.participantID = data[@"participantID"];
+      joinParam.zak =  data[@"zak"];
+      joinParam.webinarToken =  data[@"webinarToken"];
```

Comment:
1. are joining params `zak` and `webinarToken` available for android and can we use that

2. I think we should name it `zoomAccessToken` - since this naming is already being used for start meeting:
```suggestion
      joinParam.zak =  data[@"zoomAccessToken"];
      joinParam.webinarToken =  data[@"webinarToken"];
```

Reply:
`zak` naming is ok, because zoom already use it for sdk and api.
Most names I selected from IOS sdk. Nothing from my side.

Reply:
maybe I missed android support for few fields. I simply was not sure what the right one.
So I just skip some that I couldn't test. (all old params are supported)

Reply:
I understand ios uses `zak` but this is inconsistency on their side that they should fix. That's why I proposed to be consistent when we can.